cuML/src/common/cuml_api.cpp

```diff
@@ -16,6 +16,8 @@
 #include "cuML_api.h"
 
 #include <functional>
+#include <mutex>
+#include <unordered_map>
```

Comment:
I think the includes of `mutex` and `unordered_map` are not needed as both types are only indirectly used via `HandleMap`.

Reply:
True, the includes come indirectly from `cumlHandle.hpp`. 
Removed it.